<!DOCTYPE html>
<html>
<head>
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta charset="utf-8" />
</head>

<body style="margin: 0;">

<div id="p102" style="overflow: hidden; position: relative; background-color: white; width: 935px; height: 1210px;">

<!-- Begin shared CSS values -->
<style class="shared-css" type="text/css" >
.t {
	transform-origin: bottom left;
	z-index: 2;
	position: absolute;
	white-space: pre;
	overflow: visible;
	line-height: 1.5;
}
.text-container {
	white-space: pre;
}
@supports (-webkit-touch-callout: none) {
	.text-container {
		white-space: normal;
	}
}
</style>
<!-- End shared CSS values -->


<!-- Begin inline CSS -->
<style type="text/css" >

#t1_102{left:119px;bottom:1134px;letter-spacing:0.11px;word-spacing:0.02px;}
#t2_102{left:808px;bottom:1132px;letter-spacing:-0.08px;}
#t3_102{left:141px;bottom:1081px;letter-spacing:0.14px;word-spacing:0.86px;}
#t4_102{left:162px;bottom:1060px;letter-spacing:0.13px;}
#t5_102{left:162px;bottom:1025px;letter-spacing:0.13px;word-spacing:-0.46px;}
#t6_102{left:288px;bottom:1033px;}
#t7_102{left:293px;bottom:1025px;}
#t8_102{left:301px;bottom:1025px;letter-spacing:0.13px;word-spacing:-0.81px;}
#t9_102{left:434px;bottom:1033px;}
#ta_102{left:439px;bottom:1025px;letter-spacing:0.11px;word-spacing:-0.61px;}
#tb_102{left:587px;bottom:1033px;}
#tc_102{left:592px;bottom:1025px;letter-spacing:0.12px;word-spacing:-0.17px;}
#td_102{left:716px;bottom:1033px;}
#te_102{left:162px;bottom:1016px;}
#tf_102{left:167px;bottom:1008px;letter-spacing:0.11px;word-spacing:-0.52px;}
#tg_102{left:162px;bottom:998px;}
#th_102{left:170px;bottom:990px;letter-spacing:0.1px;word-spacing:-1.11px;}
#ti_102{left:666px;bottom:998px;}
#tj_102{left:673px;bottom:990px;letter-spacing:0.03px;word-spacing:-0.85px;}
#tk_102{left:162px;bottom:972px;letter-spacing:0.1px;word-spacing:-0.36px;}
#tl_102{left:162px;bottom:937px;letter-spacing:0.15px;}
#tm_102{left:243px;bottom:937px;letter-spacing:0.11px;word-spacing:2.75px;}
#tn_102{left:162px;bottom:920px;letter-spacing:0.11px;word-spacing:0.02px;}
#to_102{left:162px;bottom:902px;letter-spacing:0.11px;word-spacing:0.93px;}
#tp_102{left:162px;bottom:884px;letter-spacing:0.11px;word-spacing:0.11px;}
#tq_102{left:162px;bottom:867px;letter-spacing:0.13px;word-spacing:-0.07px;}
#tr_102{left:308px;bottom:867px;letter-spacing:0.11px;word-spacing:-0.07px;}
#ts_102{left:162px;bottom:850px;letter-spacing:0.12px;word-spacing:1.21px;}
#tt_102{left:162px;bottom:832px;letter-spacing:0.11px;word-spacing:0.33px;}
#tu_102{left:162px;bottom:814px;letter-spacing:0.1px;}
#tv_102{left:162px;bottom:797px;letter-spacing:0.02px;}
#tw_102{left:213px;bottom:797px;letter-spacing:0.04px;word-spacing:0.43px;}
#tx_102{left:162px;bottom:779px;letter-spacing:0.1px;word-spacing:0.59px;}
#ty_102{left:162px;bottom:762px;letter-spacing:0.05px;word-spacing:-1.2px;}
#tz_102{left:162px;bottom:744px;letter-spacing:0.11px;word-spacing:0.04px;}
#t10_102{left:162px;bottom:726px;letter-spacing:0.12px;word-spacing:0.63px;}
#t11_102{left:347px;bottom:726px;letter-spacing:0.1px;word-spacing:0.64px;}
#t12_102{left:162px;bottom:709px;letter-spacing:0.09px;word-spacing:0.81px;}
#t13_102{left:162px;bottom:691px;letter-spacing:0.06px;word-spacing:0.38px;}
#t14_102{left:162px;bottom:674px;letter-spacing:0.1px;word-spacing:-0.67px;}
#t15_102{left:166px;bottom:656px;letter-spacing:0.12px;word-spacing:2.2px;}
#t16_102{left:162px;bottom:639px;letter-spacing:0.1px;word-spacing:-0.21px;}
#t17_102{left:162px;bottom:621px;letter-spacing:0.14px;}
#t18_102{left:283px;bottom:621px;letter-spacing:0.12px;word-spacing:2.65px;}
#t19_102{left:162px;bottom:604px;letter-spacing:0.1px;word-spacing:0.53px;}
#t1a_102{left:162px;bottom:586px;letter-spacing:0.11px;word-spacing:-0.32px;}
#t1b_102{left:162px;bottom:568px;letter-spacing:0.11px;word-spacing:0.11px;}
#t1c_102{left:141px;bottom:512px;letter-spacing:0.14px;}
#t1d_102{left:395px;bottom:513px;}
#t1e_102{left:406px;bottom:512px;letter-spacing:0.14px;}
#t1f_102{left:162px;bottom:491px;letter-spacing:0.15px;}
#t1g_102{left:162px;bottom:456px;letter-spacing:0.07px;word-spacing:2.87px;}
#t1h_102{left:281px;bottom:464px;}
#t1i_102{left:286px;bottom:456px;letter-spacing:0.04px;word-spacing:2.87px;}
#t1j_102{left:390px;bottom:464px;}
#t1k_102{left:395px;bottom:456px;letter-spacing:0.07px;word-spacing:2.84px;}
#t1l_102{left:468px;bottom:464px;}
#t1m_102{left:480px;bottom:456px;letter-spacing:0.12px;word-spacing:2.92px;}
#t1n_102{left:643px;bottom:464px;}
#t1o_102{left:654px;bottom:464px;}
#t1p_102{left:659px;bottom:456px;letter-spacing:0.12px;word-spacing:2.91px;}
#t1q_102{left:162px;bottom:438px;letter-spacing:0.11px;word-spacing:3.86px;}
#t1r_102{left:283px;bottom:446px;}
#t1s_102{left:288px;bottom:438px;letter-spacing:0.12px;word-spacing:3.13px;}
#t1t_102{left:162px;bottom:421px;letter-spacing:0.11px;word-spacing:-0.05px;}
#t1u_102{left:162px;bottom:386px;letter-spacing:0.11px;word-spacing:2.25px;}
#t1v_102{left:162px;bottom:367px;letter-spacing:0.11px;word-spacing:0.37px;}
#t1w_102{left:162px;bottom:349px;letter-spacing:0.08px;word-spacing:-0.55px;}
#t1x_102{left:162px;bottom:330px;letter-spacing:0.1px;word-spacing:-0.99px;}
#t1y_102{left:162px;bottom:312px;letter-spacing:0.1px;word-spacing:0.16px;}
#t1z_102{left:162px;bottom:293px;letter-spacing:0.1px;word-spacing:-0.61px;}
#t20_102{left:162px;bottom:276px;letter-spacing:0.08px;word-spacing:0.94px;}
#t21_102{left:425px;bottom:276px;letter-spacing:0.01px;word-spacing:0.99px;}
#t22_102{left:474px;bottom:276px;letter-spacing:0.09px;word-spacing:0.86px;}
#t23_102{left:162px;bottom:257px;letter-spacing:0.1px;word-spacing:-0.69px;}
#t24_102{left:162px;bottom:239px;letter-spacing:0.12px;word-spacing:3.19px;}
#t25_102{left:162px;bottom:222px;letter-spacing:0.12px;word-spacing:2.77px;}
#t26_102{left:162px;bottom:202px;letter-spacing:0.1px;word-spacing:-0.4px;}
#t27_102{left:162px;bottom:183px;letter-spacing:0.09px;word-spacing:1.38px;}
#t28_102{left:162px;bottom:164px;letter-spacing:0.11px;word-spacing:-0.45px;}
#t29_102{left:162px;bottom:145px;letter-spacing:0.11px;word-spacing:0.38px;}
#t2a_102{left:162px;bottom:127px;letter-spacing:0.11px;word-spacing:0.57px;}
#t2b_102{left:162px;bottom:108px;letter-spacing:0.12px;word-spacing:3.02px;}
#t2c_102{left:162px;bottom:89px;letter-spacing:0.12px;word-spacing:3.87px;}

.s0_102{font-size:15px;font-family:TimesNewRomanPSMT_6e;color:#000;}
.s1_102{font-size:17px;font-family:Calibri_7k;color:#000;}
.s2_102{font-size:18px;font-family:TimesNewRomanPS-BoldMT_7w;color:#000;}
.s3_102{font-size:15px;font-family:TimesNewRomanPS-BoldMT_7s;color:#000;}
.s4_102{font-size:10px;font-family:TimesNewRomanPS-BoldMT_8j;color:#000;}
.s5_102{font-size:10px;font-family:TimesNewRomanPSMT_87;color:#000;}
.s6_102{font-size:12px;font-family:TimesNewRomanPS-BoldMT_c7;color:#000;}
.s7_102{font-size:15px;font-family:TimesNewRomanPS-ItalicMT_8n;color:#000;}
</style>
<!-- End inline CSS -->

<!-- Begin embedded font definitions -->
<style id="fonts102" type="text/css" >

@font-face {
	font-family: Calibri_7k;
	src: url("fonts/Calibri_7k.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7s;
	src: url("fonts/TimesNewRomanPS-BoldMT_7s.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7w;
	src: url("fonts/TimesNewRomanPS-BoldMT_7w.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_8j;
	src: url("fonts/TimesNewRomanPS-BoldMT_8j.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_c7;
	src: url("fonts/TimesNewRomanPS-BoldMT_c7.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-ItalicMT_8n;
	src: url("fonts/TimesNewRomanPS-ItalicMT_8n.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_6e;
	src: url("fonts/TimesNewRomanPSMT_6e.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_87;
	src: url("fonts/TimesNewRomanPSMT_87.woff") format("woff");
}

</style>
<!-- End embedded font definitions -->

<!-- Begin page background -->
<div id="pg102Overlay" style="width:100%; height:100%; position:absolute; z-index:1; background-color:rgba(0,0,0,0); -webkit-user-select: none;"></div>
<div id="pg102" style="-webkit-user-select: none;"><object width="935" height="1210" data="102/102.svg" type="image/svg+xml" id="pdf102" style="width:935px; height:1210px; -moz-transform:scale(1); z-index: 0;"></object></div>
<!-- End page background -->


<!-- Begin text definitions (Positioned/styled in CSS) -->
<div class="text-container"><span id="t1_102" class="t s0_102">Brain: Chemistry to Cognition </span><span id="t2_102" class="t s1_102">94 </span>
<span id="t3_102" class="t s2_102">P19: Early detection of Parkinson's disease by using artificial intelligence and analysis </span>
<span id="t4_102" class="t s2_102">of clinico-genetic correlates </span>
<span id="t5_102" class="t s3_102">Arvind Kumar Das </span>
<span id="t6_102" class="t s4_102">1 </span>
<span id="t7_102" class="t s3_102">, </span><span id="t8_102" class="t s0_102">Gyaneshwar Chaubey </span>
<span id="t9_102" class="t s5_102">2 </span>
<span id="ta_102" class="t s0_102">, Kaushal Kishor Shukla </span>
<span id="tb_102" class="t s5_102">3 </span>
<span id="tc_102" class="t s0_102">, Vijaya NathMishra </span>
<span id="td_102" class="t s5_102">1 </span>
<span id="te_102" class="t s5_102">1 </span>
<span id="tf_102" class="t s0_102">Department of Neurology, Institute of Medical Sciences, Banaras Hindu University,Varanasi </span>
<span id="tg_102" class="t s5_102">2 </span>
<span id="th_102" class="t s0_102">Department of Zoology, Institute of Science, Banaras Hindu University, Varanasi; </span>
<span id="ti_102" class="t s5_102">3 </span>
<span id="tj_102" class="t s0_102">Department of Computer </span>
<span id="tk_102" class="t s0_102">Science, Indian Institute of Technology, Banaras HinduUniversity, Varanasi </span>
<span id="tl_102" class="t s3_102">Background</span><span id="tm_102" class="t s0_102">: Parkinson's disease, a complicated neurodegenerative disorder, affects millions of people </span>
<span id="tn_102" class="t s0_102">worldwide. It influenced by hereditary variables such single nucleotide polymorphisms (SNPs), which has a </span>
<span id="to_102" class="t s0_102">negative impact on quality of life. An artificial intelligence-driven methodologies aid in identifying SNPs, </span>
<span id="tp_102" class="t s0_102">discovering novel player biomarkers, and creating prediction models for disease risk assessment. </span>
<span id="tq_102" class="t s3_102">Material and Methods</span><span id="tr_102" class="t s0_102">: Videos and blood sample of PD patients were collected from S.S.L. Hospital, IMS, </span>
<span id="ts_102" class="t s0_102">BHU, Varanasi. The AI technique was used to screening and feature selection for PD using Machine and </span>
<span id="tt_102" class="t s0_102">Deep Learning, such as KNN, SVM, NB, DT, and RF. Further genetic analysis is performed to find out the </span>
<span id="tu_102" class="t s0_102">SNPs. </span>
<span id="tv_102" class="t s3_102">Result: </span><span id="tw_102" class="t s0_102">AI tools has identified the pattern of disease (motor and non-motor function) and checkthe accuracy </span>
<span id="tx_102" class="t s0_102">rate and performance. Exome sequencing retrieved the SNPs in certain gene suchas: SNCA, PARKN, DJ1, </span>
<span id="ty_102" class="t s0_102">and LRRK2 that could potentially be used as biomarkers for the earlydiagnosis of PD. AI-driven genetic dataset </span>
<span id="tz_102" class="t s0_102">analysis uncovers genetic changes related to Parkinson's subtypes and symptoms severity. </span>
<span id="t10_102" class="t s3_102">Discussion and Conclusion: </span><span id="t11_102" class="t s0_102">AI is still in its infancy, its integration into PD research has greatpromise for </span>
<span id="t12_102" class="t s0_102">accelerating the development of novel therapies, improving patient outcomes, andultimately advancing our </span>
<span id="t13_102" class="t s0_102">understanding of the disease's core causes. In conclusion, the AI couldbe helpful in examining if any person </span>
<span id="t14_102" class="t s0_102">having any of these symptoms is a Patient of Parkinson’s </span>
<span id="t15_102" class="t s0_102">by machine learning which can detect the changes in expression and movement of the patientand their </span>
<span id="t16_102" class="t s0_102">correlation with SNPs. </span>
<span id="t17_102" class="t s3_102">Acknowledgement</span><span id="t18_102" class="t s0_102">: The authors kindly acknowledge Department of Computer Sciences, IIT, BHU for </span>
<span id="t19_102" class="t s0_102">providing software and the computational facilities for AI analysis, and Department of Zoology, Instituteof </span>
<span id="t1a_102" class="t s0_102">Science, BHU for providing plate form for genomic experiment. We are highly grateful to the Department of </span>
<span id="t1b_102" class="t s0_102">Neurology, IMS, BHU for extending laboratory facilities. </span>
<span id="t1c_102" class="t s2_102">P20: Identification of a novel Hv</span><span id="t1d_102" class="t s6_102">1 </span><span id="t1e_102" class="t s2_102">proton channel inhibitor to ameliorate </span>
<span id="t1f_102" class="t s2_102">neuroinflammation </span>
<span id="t1g_102" class="t s3_102">Ashutosh Sharma </span>
<span id="t1h_102" class="t s4_102">1 </span>
<span id="t1i_102" class="t s0_102">, Suchitra Gupta </span>
<span id="t1j_102" class="t s5_102">2 </span>
<span id="t1k_102" class="t s0_102">, Atul Goel </span>
<span id="t1l_102" class="t s5_102">2 </span>
<span id="t1m_102" class="t s0_102">&amp; Aravind Singh Kshatri </span>
<span id="t1n_102" class="t s5_102">1 </span><span id="t1o_102" class="t s5_102">1 </span>
<span id="t1p_102" class="t s0_102">Neuroscience and Ageing </span>
<span id="t1q_102" class="t s0_102">Biology Division, </span>
<span id="t1r_102" class="t s5_102">2 </span>
<span id="t1s_102" class="t s0_102">Medicinal and Process Chemistry Division CSIR- Central Drug Research Institute, </span>
<span id="t1t_102" class="t s0_102">Lucknow, India </span>
<span id="t1u_102" class="t s0_102">Voltage-gated proton channels (Hv1) are specifically expressed in the microglia of mammalian nervous </span>
<span id="t1v_102" class="t s0_102">systems and are associated with neuroinflammation. Emerging data from various preclinical animal models </span>
<span id="t1w_102" class="t s0_102">demonstrate that persistent activation of these channels significantly contribute to chronic neuroinflammation </span>
<span id="t1x_102" class="t s0_102">and subsequentlyneurodegeneration. Despite the potential clinical significance, no high throughput assay Hv1 </span>
<span id="t1y_102" class="t s0_102">inhibitors with a reasonable affinity and selectivity currently exist to target these channels. To address these </span>
<span id="t1z_102" class="t s0_102">research gaps, we aimed to identify new classes ofpotent and selective inhibitors of this channel using a novel </span>
<span id="t20_102" class="t s0_102">fluorescent based HTS methodology. The </span><span id="t21_102" class="t s7_102">in vitro </span><span id="t22_102" class="t s0_102">efficacy of the identified hit compounds was done using </span>
<span id="t23_102" class="t s0_102">RT-PCR technique and docking studies were also performed to identify their binding sitesin the Hv1 channel. </span>
<span id="t24_102" class="t s0_102">After screening of an in-house library of heterocyclic compounds, we have identified a benzothiazole </span>
<span id="t25_102" class="t s0_102">derivative, BTZ-26 as a potent inhibitor of Hv1 channels (IC50=0.3µM). Subsequently, our specificity </span>
<span id="t26_102" class="t s0_102">studies established that it does not alter the activity of other ion channels belonging to Nav and TRP families. </span>
<span id="t27_102" class="t s0_102">Cell toxicity experiments revealed that BTZ-26 is non-toxic (up to 10µM) for BV-2 microglial cells. Our </span>
<span id="t28_102" class="t s0_102">preliminary data also indicated that BTZ-26 largely abrogated inflammatoryneurotoxicity by reprogramming </span>
<span id="t29_102" class="t s0_102">LPS-activated microglia. Molecular docking data indicated that BTZ-26 binds to the extracellular region of </span>
<span id="t2a_102" class="t s0_102">Hv1 channel and makes contact with hydrophobic amino acids. The predicted binding region of BTZ-26 is </span>
<span id="t2b_102" class="t s0_102">completely different than that of the existing modest Hv1 channel blocker (2GBI) which binds in the </span>
<span id="t2c_102" class="t s0_102">intracellular end of the channel. Although our hit appears to be Hv1 selective, a through functional </span></div>
<!-- End text definitions -->


</div>
</body>
</html>
