<!DOCTYPE html>
<html>
<head>
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta charset="utf-8" />
</head>

<body style="margin: 0;">

<div id="p155" style="overflow: hidden; position: relative; background-color: white; width: 935px; height: 1210px;">

<!-- Begin shared CSS values -->
<style class="shared-css" type="text/css" >
.t {
	transform-origin: bottom left;
	z-index: 2;
	position: absolute;
	white-space: pre;
	overflow: visible;
	line-height: 1.5;
}
.text-container {
	white-space: pre;
}
@supports (-webkit-touch-callout: none) {
	.text-container {
		white-space: normal;
	}
}
</style>
<!-- End shared CSS values -->


<!-- Begin inline CSS -->
<style type="text/css" >

#t1_155{left:119px;bottom:1134px;letter-spacing:0.11px;word-spacing:0.02px;}
#t2_155{left:799px;bottom:1132px;letter-spacing:-0.08px;}
#t3_155{left:141px;bottom:1041px;letter-spacing:0.14px;word-spacing:2.34px;}
#t4_155{left:162px;bottom:1020px;letter-spacing:0.15px;word-spacing:-0.06px;}
#t5_155{left:162px;bottom:983px;letter-spacing:0.13px;word-spacing:-0.05px;}
#t6_155{left:255px;bottom:991px;}
#t7_155{left:260px;bottom:983px;letter-spacing:0.12px;word-spacing:-0.05px;}
#t8_155{left:403px;bottom:991px;}
#t9_155{left:408px;bottom:983px;letter-spacing:0.12px;word-spacing:-0.02px;}
#ta_155{left:523px;bottom:991px;}
#tb_155{left:528px;bottom:983px;letter-spacing:0.11px;word-spacing:-0.08px;}
#tc_155{left:658px;bottom:991px;}
#td_155{left:663px;bottom:983px;}
#te_155{left:162px;bottom:965px;letter-spacing:0.13px;word-spacing:-0.05px;}
#tf_155{left:265px;bottom:973px;}
#tg_155{left:162px;bottom:955px;}
#th_155{left:167px;bottom:948px;letter-spacing:0.11px;word-spacing:1.27px;}
#ti_155{left:162px;bottom:930px;letter-spacing:0.08px;}
#tj_155{left:202px;bottom:938px;}
#tk_155{left:207px;bottom:930px;letter-spacing:0.11px;word-spacing:-0.1px;}
#tl_155{left:162px;bottom:912px;letter-spacing:0.11px;word-spacing:-0.05px;}
#tm_155{left:162px;bottom:877px;letter-spacing:0.12px;}
#tn_155{left:252px;bottom:877px;letter-spacing:0.1px;word-spacing:-0.71px;}
#to_155{left:162px;bottom:860px;letter-spacing:0.12px;word-spacing:2.14px;}
#tp_155{left:162px;bottom:842px;letter-spacing:0.11px;word-spacing:0.04px;}
#tq_155{left:362px;bottom:842px;}
#tr_155{left:370px;bottom:842px;letter-spacing:0.11px;word-spacing:0.01px;}
#ts_155{left:162px;bottom:824px;letter-spacing:0.12px;word-spacing:0.06px;}
#tt_155{left:426px;bottom:824px;}
#tu_155{left:162px;bottom:807px;letter-spacing:0.13px;word-spacing:0.99px;}
#tv_155{left:304px;bottom:807px;letter-spacing:0.11px;word-spacing:0.76px;}
#tw_155{left:162px;bottom:789px;letter-spacing:0.11px;word-spacing:0.77px;}
#tx_155{left:162px;bottom:772px;letter-spacing:0.11px;word-spacing:0.65px;}
#ty_155{left:162px;bottom:754px;letter-spacing:0.1px;word-spacing:0.01px;}
#tz_155{left:162px;bottom:737px;letter-spacing:0.12px;}
#t10_155{left:210px;bottom:737px;letter-spacing:0.11px;word-spacing:0.89px;}
#t11_155{left:568px;bottom:737px;letter-spacing:0.11px;}
#t12_155{left:608px;bottom:737px;letter-spacing:0.1px;word-spacing:0.85px;}
#t13_155{left:698px;bottom:737px;letter-spacing:0.12px;}
#t14_155{left:733px;bottom:737px;letter-spacing:0.1px;word-spacing:0.75px;}
#t15_155{left:162px;bottom:719px;letter-spacing:0.09px;word-spacing:-0.03px;}
#t16_155{left:304px;bottom:719px;letter-spacing:0.11px;}
#t17_155{left:378px;bottom:719px;letter-spacing:0.09px;}
#t18_155{left:404px;bottom:719px;letter-spacing:0.11px;}
#t19_155{left:486px;bottom:719px;letter-spacing:0.1px;word-spacing:-0.15px;}
#t1a_155{left:162px;bottom:701px;letter-spacing:0.11px;word-spacing:-0.62px;}
#t1b_155{left:162px;bottom:684px;letter-spacing:0.1px;word-spacing:-0.5px;}
#t1c_155{left:703px;bottom:684px;letter-spacing:0.13px;}
#t1d_155{left:746px;bottom:684px;letter-spacing:0.11px;}
#t1e_155{left:162px;bottom:666px;letter-spacing:0.11px;word-spacing:-0.6px;}
#t1f_155{left:162px;bottom:649px;letter-spacing:0.13px;word-spacing:3.1px;}
#t1g_155{left:349px;bottom:649px;letter-spacing:0.11px;word-spacing:3.21px;}
#t1h_155{left:635px;bottom:649px;letter-spacing:0.11px;}
#t1i_155{left:678px;bottom:649px;letter-spacing:0.11px;word-spacing:3.2px;}
#t1j_155{left:775px;bottom:649px;letter-spacing:0.12px;}
#t1k_155{left:813px;bottom:649px;letter-spacing:0.05px;}
#t1l_155{left:162px;bottom:631px;letter-spacing:0.11px;word-spacing:0.68px;}
#t1m_155{left:162px;bottom:613px;letter-spacing:0.1px;}
#t1n_155{left:220px;bottom:613px;letter-spacing:0.18px;}
#t1o_155{left:260px;bottom:613px;letter-spacing:0.1px;word-spacing:-0.96px;}
#t1p_155{left:162px;bottom:596px;letter-spacing:0.1px;word-spacing:-0.18px;}
#t1q_155{left:162px;bottom:578px;letter-spacing:0.1px;word-spacing:-0.01px;}
#t1r_155{left:162px;bottom:561px;letter-spacing:0.14px;}
#t1s_155{left:299px;bottom:561px;letter-spacing:0.11px;word-spacing:0.75px;}
#t1t_155{left:162px;bottom:543px;letter-spacing:0.12px;word-spacing:3.77px;}
#t1u_155{left:162px;bottom:526px;letter-spacing:0.1px;}
#t1v_155{left:223px;bottom:526px;letter-spacing:0.1px;word-spacing:1.64px;}
#t1w_155{left:352px;bottom:526px;letter-spacing:0.12px;}
#t1x_155{left:436px;bottom:526px;letter-spacing:0.12px;word-spacing:0.64px;}
#t1y_155{left:162px;bottom:508px;letter-spacing:0.12px;word-spacing:2.24px;}
#t1z_155{left:141px;bottom:457px;letter-spacing:0.17px;word-spacing:7.31px;}
#t20_155{left:162px;bottom:437px;letter-spacing:0.14px;word-spacing:-0.08px;}
#t21_155{left:162px;bottom:396px;letter-spacing:0.12px;word-spacing:-0.65px;}
#t22_155{left:286px;bottom:396px;letter-spacing:0.12px;word-spacing:-0.73px;}
#t23_155{left:162px;bottom:378px;letter-spacing:0.14px;}
#t24_155{left:205px;bottom:378px;letter-spacing:0.11px;word-spacing:0.02px;}
#t25_155{left:162px;bottom:348px;letter-spacing:0.12px;}
#t26_155{left:257px;bottom:348px;letter-spacing:0.12px;word-spacing:4.78px;}
#t27_155{left:162px;bottom:331px;letter-spacing:0.1px;word-spacing:-0.79px;}
#t28_155{left:162px;bottom:313px;letter-spacing:0.1px;word-spacing:-0.9px;}
#t29_155{left:162px;bottom:296px;letter-spacing:0.12px;word-spacing:1.67px;}
#t2a_155{left:162px;bottom:278px;letter-spacing:0.1px;word-spacing:0.75px;}
#t2b_155{left:162px;bottom:261px;letter-spacing:0.12px;}
#t2c_155{left:162px;bottom:243px;letter-spacing:0.11px;}
#t2d_155{left:229px;bottom:243px;letter-spacing:0.11px;word-spacing:0.7px;}
#t2e_155{left:162px;bottom:225px;letter-spacing:0.11px;word-spacing:1.37px;}
#t2f_155{left:162px;bottom:208px;letter-spacing:0.11px;word-spacing:0.78px;}
#t2g_155{left:162px;bottom:190px;letter-spacing:0.11px;word-spacing:0.02px;}
#t2h_155{left:774px;bottom:190px;}
#t2i_155{left:162px;bottom:173px;letter-spacing:0.12px;}
#t2j_155{left:214px;bottom:173px;letter-spacing:0.11px;word-spacing:1.06px;}
#t2k_155{left:162px;bottom:155px;letter-spacing:0.12px;word-spacing:3.45px;}
#t2l_155{left:162px;bottom:138px;letter-spacing:0.1px;word-spacing:-0.53px;}
#t2m_155{left:162px;bottom:120px;letter-spacing:0.12px;word-spacing:2.45px;}
#t2n_155{left:162px;bottom:102px;letter-spacing:0.11px;word-spacing:0.17px;}
#t2o_155{left:162px;bottom:85px;letter-spacing:0.11px;word-spacing:-0.09px;}
#t2p_155{left:589px;bottom:85px;}

.s0_155{font-size:15px;font-family:TimesNewRomanPSMT_6e;color:#000;}
.s1_155{font-size:17px;font-family:Calibri_7k;color:#000;}
.s2_155{font-size:18px;font-family:TimesNewRomanPS-BoldMT_7w;color:#000;}
.s3_155{font-size:15px;font-family:TimesNewRomanPS-BoldMT_7s;color:#000;}
.s4_155{font-size:10px;font-family:TimesNewRomanPS-BoldMT_8j;color:#000;}
.s5_155{font-size:10px;font-family:TimesNewRomanPSMT_87;color:#000;}
.s6_155{font-size:15px;font-family:TimesNewRomanPS-ItalicMT_8n;color:#000;}
</style>
<!-- End inline CSS -->

<!-- Begin embedded font definitions -->
<style id="fonts155" type="text/css" >

@font-face {
	font-family: Calibri_7k;
	src: url("fonts/Calibri_7k.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7s;
	src: url("fonts/TimesNewRomanPS-BoldMT_7s.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7w;
	src: url("fonts/TimesNewRomanPS-BoldMT_7w.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_8j;
	src: url("fonts/TimesNewRomanPS-BoldMT_8j.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-ItalicMT_8n;
	src: url("fonts/TimesNewRomanPS-ItalicMT_8n.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_6e;
	src: url("fonts/TimesNewRomanPSMT_6e.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_87;
	src: url("fonts/TimesNewRomanPSMT_87.woff") format("woff");
}

</style>
<!-- End embedded font definitions -->

<!-- Begin page background -->
<div id="pg155Overlay" style="width:100%; height:100%; position:absolute; z-index:1; background-color:rgba(0,0,0,0); -webkit-user-select: none;"></div>
<div id="pg155" style="-webkit-user-select: none;"><object width="935" height="1210" data="155/155.svg" type="image/svg+xml" id="pdf155" style="width:935px; height:1210px; -moz-transform:scale(1); z-index: 0;"></object></div>
<!-- End page background -->


<!-- Begin text definitions (Positioned/styled in CSS) -->
<div class="text-container"><span id="t1_155" class="t s0_155">Brain: Chemistry to Cognition </span><span id="t2_155" class="t s1_155">147 </span>
<span id="t3_155" class="t s2_155">P120: Identification and association of genetic factors: a cohort study of Post-Stroke </span>
<span id="t4_155" class="t s2_155">Depression among eastern Indian population </span>
<span id="t5_155" class="t s3_155">Smriti Mishra </span>
<span id="t6_155" class="t s4_155">1 </span>
<span id="t7_155" class="t s0_155">, Dipanwita Sadhukhan </span>
<span id="t8_155" class="t s5_155">1 </span>
<span id="t9_155" class="t s0_155">, Biman Kanti Roy </span>
<span id="ta_155" class="t s5_155">2 </span>
<span id="tb_155" class="t s0_155">, Subhra Prakash Hui </span>
<span id="tc_155" class="t s5_155">3 </span>
<span id="td_155" class="t s0_155">, </span>
<span id="te_155" class="t s0_155">Arindam Biswas </span>
<span id="tf_155" class="t s5_155">1 </span>
<span id="tg_155" class="t s5_155">1 </span>
<span id="th_155" class="t s0_155">Molecular Biology &amp; Clinical Neuroscience Division, National Neurosciences Centre, Calcutta, Kolkata, </span>
<span id="ti_155" class="t s0_155">India; </span>
<span id="tj_155" class="t s5_155">2 </span>
<span id="tk_155" class="t s0_155">Bangur Institute of Neurosciences, IPGME&amp;R, Kolkata, India; </span>
<span id="tl_155" class="t s0_155">S.N. Pradhan Centre for Neurosciences, University of Calcutta. </span>
<span id="tm_155" class="t s3_155">Background: </span><span id="tn_155" class="t s0_155">Post-stroke depression (PSD) is associated with increased morbidity and mortality. In Kolkata, </span>
<span id="to_155" class="t s0_155">the prevalence of PSD was 36.98% and ~17% developed depression annually. PSD is linked to worsen </span>
<span id="tp_155" class="t s0_155">cognitive and physical outcomes</span><span id="tq_155" class="t s3_155">. </span><span id="tr_155" class="t s0_155">In this study, we aim to determine the association of polymorphic variants </span>
<span id="ts_155" class="t s0_155">with PSD among eastern Indian population</span><span id="tt_155" class="t s3_155">. </span>
<span id="tu_155" class="t s3_155">Materials &amp; Methods</span><span id="tv_155" class="t s0_155">: The patient cohort consisted of 91 Ischemic stroke and 68 ICH cases from eastern </span>
<span id="tw_155" class="t s0_155">India. PCR-RFLP techniques was used to genotype the polymorphisms, based on associated literature. We </span>
<span id="tx_155" class="t s0_155">used Geriatric Depression Scale (GDS) for assessment of depression. Statistical calculation was performed </span>
<span id="ty_155" class="t s0_155">using Mann-Whitney U test calculator, on line web tool. </span>
<span id="tz_155" class="t s3_155">Results</span><span id="t10_155" class="t s0_155">: Among the studied polymorphic variants, rs6559833 of </span><span id="t11_155" class="t s6_155">TRKβ </span><span id="t12_155" class="t s0_155">and rs6159 of </span><span id="t13_155" class="t s6_155">CRH </span><span id="t14_155" class="t s0_155">are found tobe </span>
<span id="t15_155" class="t s0_155">statistically significant </span><span id="t16_155" class="t s3_155">(P=0.0455) </span><span id="t17_155" class="t s0_155">and </span><span id="t18_155" class="t s3_155">(P=0.01994) </span><span id="t19_155" class="t s0_155">respectively; with PSD in ischemic stroke patients. The </span>
<span id="t1a_155" class="t s0_155">mean GDS score for the “TT” survivors of rs6559833 was 14.45±6.36 andthe score for the “AA” survivors of </span>
<span id="t1b_155" class="t s0_155">rs6159 was 6.10 ± 2.02. Along with it, a trend has been observed in the variant, rs6265 of </span><span id="t1c_155" class="t s6_155">BDNF </span><span id="t1d_155" class="t s0_155">(P=0.08124) </span>
<span id="t1e_155" class="t s0_155">among post-stroke depressed individualsof ischemic stroke. </span>
<span id="t1f_155" class="t s3_155">Discussion and Conclusions</span><span id="t1g_155" class="t s0_155">: A significant association of rs6559833 of </span><span id="t1h_155" class="t s6_155">TRKβ </span><span id="t1i_155" class="t s0_155">and rs6159 of </span><span id="t1j_155" class="t s6_155">CRH </span><span id="t1k_155" class="t s0_155">is </span>
<span id="t1l_155" class="t s0_155">observed with PSD among ethnic Bengali population of eastern India and a like-wise trend is observed for </span>
<span id="t1m_155" class="t s0_155">rs6265of </span><span id="t1n_155" class="t s6_155">BDNF</span><span id="t1o_155" class="t s0_155">. To best of our knowledge, this is first report from India, identifying the association of genetic </span>
<span id="t1p_155" class="t s0_155">variants with PSD. In future, a large number of samples from other ethnic populationneeds to be validated to </span>
<span id="t1q_155" class="t s0_155">understand the genetic pathobiology of depression after stroke. </span>
<span id="t1r_155" class="t s3_155">Acknowledgements: </span><span id="t1s_155" class="t s0_155">The authors thank the patient and the family members who participated in the study. </span>
<span id="t1t_155" class="t s0_155">Supported by grants from the Department of Science &amp; Technology, Govt. of India, underCognitive </span>
<span id="t1u_155" class="t s0_155">Science </span><span id="t1v_155" class="t s0_155">Research Initiative </span><span id="t1w_155" class="t s0_155">Programme </span><span id="t1x_155" class="t s0_155">to author Dipanwita Sadhukhan(DST/CSRI-PDF/2021/12) and </span>
<span id="t1y_155" class="t s0_155">the corresponding author Arindam Biswas (DST/CSRI-P/2017/22). </span>
<span id="t1z_155" class="t s2_155">P121: Mouse brain development: Decoding developmental neurobiology through </span>
<span id="t20_155" class="t s2_155">integrated multi-omics analyses </span>
<span id="t21_155" class="t s3_155">Sneha Manjunath, </span><span id="t22_155" class="t s0_155">Vatsal Mehra, Manojkumar, Murray Blackmore and Ishwariya Venkatesh </span>
<span id="t23_155" class="t s0_155">CSIR- </span><span id="t24_155" class="t s0_155">Centre for Cellular and Molecular Biology, Hyderabad </span>
<span id="t25_155" class="t s3_155">Background: </span><span id="t26_155" class="t s0_155">Understanding the regulatory landscape of the developing mouse brain is crucial for </span>
<span id="t27_155" class="t s0_155">unravelling the intricate processes driving brain development and function. However, analysing all regulatory </span>
<span id="t28_155" class="t s0_155">layers simultaneously poses challenges for bioinformaticians, including the management and analysis of large </span>
<span id="t29_155" class="t s0_155">and complex datasets generated by high-throughput sequencing techniques. This study introduces Mouse </span>
<span id="t2a_155" class="t s0_155">brain dev, an innovative user-interactive website designed to retrieve regulatory information in the field of </span>
<span id="t2b_155" class="t s0_155">neurobiology. </span>
<span id="t2c_155" class="t s3_155">Methods: </span><span id="t2d_155" class="t s0_155">The Mouse brain dev database integrates diverse datasets, including gene expression, chromatin </span>
<span id="t2e_155" class="t s0_155">accessibility, transcription factor binding, and transcription factor footprints, obtained from the ENCODE </span>
<span id="t2f_155" class="t s0_155">consortia across multiple developmental time points. This comprehensive resource enables neurobiologists </span>
<span id="t2g_155" class="t s0_155">to explore intricate regulatory networks and uncover the molecular basis of complex brain functions</span><span id="t2h_155" class="t s3_155">. </span>
<span id="t2i_155" class="t s3_155">Result: </span><span id="t2j_155" class="t s0_155">Mouse brain dev’s unique feature is its intuitive graphical user interface (GUI), which overcomes </span>
<span id="t2k_155" class="t s0_155">the constraint of effectively visualising data and analysis. The GUI provides interactive and dynamic </span>
<span id="t2l_155" class="t s0_155">visualisations, facilitating the interpretation of complex regulatory patterns and relationships. This empowers </span>
<span id="t2m_155" class="t s0_155">researchers to identify trends, patterns, and potential regulatory interactions that may not beapparent in </span>
<span id="t2n_155" class="t s0_155">traditional formats. By addressing the challenges, Mouse brain dev offers a holistic approachto studying the </span>
<span id="t2o_155" class="t s0_155">dynamic regulatory landscape of the mouse brain during development</span><span id="t2p_155" class="t s3_155">. </span></div>
<!-- End text definitions -->


</div>
</body>
</html>
