<!DOCTYPE html>
<html>
<head>
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta charset="utf-8" />
</head>

<body style="margin: 0;">

<div id="p73" style="overflow: hidden; position: relative; background-color: white; width: 935px; height: 1210px;">

<!-- Begin shared CSS values -->
<style class="shared-css" type="text/css" >
.t {
	transform-origin: bottom left;
	z-index: 2;
	position: absolute;
	white-space: pre;
	overflow: visible;
	line-height: 1.5;
}
.text-container {
	white-space: pre;
}
@supports (-webkit-touch-callout: none) {
	.text-container {
		white-space: normal;
	}
}
</style>
<!-- End shared CSS values -->


<!-- Begin inline CSS -->
<style type="text/css" >

#t1_73{left:119px;bottom:1134px;letter-spacing:0.11px;word-spacing:0.02px;}
#t2_73{left:808px;bottom:1132px;letter-spacing:-0.08px;}
#t3_73{left:141px;bottom:1081px;letter-spacing:0.14px;word-spacing:-0.65px;}
#t4_73{left:162px;bottom:1059px;letter-spacing:0.14px;}
#t5_73{left:162px;bottom:1021px;letter-spacing:0.12px;word-spacing:0.04px;}
#t6_73{left:162px;bottom:1004px;letter-spacing:0.11px;word-spacing:0.03px;}
#t7_73{left:162px;bottom:986px;letter-spacing:0.11px;}
#t8_73{left:162px;bottom:952px;letter-spacing:-0.21px;word-spacing:4.05px;}
#t9_73{left:162px;bottom:935px;letter-spacing:-0.19px;word-spacing:0.77px;}
#ta_73{left:162px;bottom:919px;letter-spacing:-0.2px;word-spacing:0.06px;}
#tb_73{left:162px;bottom:902px;letter-spacing:-0.19px;word-spacing:-0.35px;}
#tc_73{left:162px;bottom:886px;letter-spacing:-0.2px;word-spacing:-0.47px;}
#td_73{left:162px;bottom:869px;letter-spacing:-0.22px;word-spacing:1.27px;}
#te_73{left:162px;bottom:852px;letter-spacing:-0.2px;word-spacing:0.02px;}
#tf_73{left:162px;bottom:835px;letter-spacing:-0.21px;word-spacing:2.63px;}
#tg_73{left:162px;bottom:819px;letter-spacing:-0.2px;word-spacing:1.31px;}
#th_73{left:162px;bottom:802px;letter-spacing:-0.2px;word-spacing:-0.12px;}
#ti_73{left:162px;bottom:785px;letter-spacing:-0.2px;word-spacing:0.56px;}
#tj_73{left:162px;bottom:769px;letter-spacing:-0.19px;word-spacing:0.55px;}
#tk_73{left:162px;bottom:752px;letter-spacing:-0.2px;word-spacing:0.15px;}
#tl_73{left:162px;bottom:735px;letter-spacing:-0.21px;word-spacing:4.45px;}
#tm_73{left:162px;bottom:719px;letter-spacing:-0.21px;word-spacing:1.93px;}
#tn_73{left:162px;bottom:702px;letter-spacing:-0.2px;word-spacing:0.14px;}
#to_73{left:162px;bottom:685px;letter-spacing:-0.2px;word-spacing:1.37px;}
#tp_73{left:162px;bottom:668px;letter-spacing:-0.2px;word-spacing:0.8px;}
#tq_73{left:162px;bottom:652px;letter-spacing:-0.22px;word-spacing:2.41px;}
#tr_73{left:162px;bottom:635px;letter-spacing:-0.21px;word-spacing:2.02px;}
#ts_73{left:162px;bottom:619px;letter-spacing:-0.22px;word-spacing:2.29px;}
#tt_73{left:162px;bottom:602px;letter-spacing:-0.21px;word-spacing:0.4px;}
#tu_73{left:162px;bottom:585px;letter-spacing:-0.19px;word-spacing:-0.09px;}
#tv_73{left:162px;bottom:568px;letter-spacing:-0.2px;word-spacing:2.62px;}
#tw_73{left:162px;bottom:552px;letter-spacing:-0.19px;word-spacing:0.02px;}
#tx_73{left:162px;bottom:535px;letter-spacing:-0.2px;word-spacing:-0.24px;}
#ty_73{left:162px;bottom:518px;letter-spacing:-0.2px;word-spacing:3.16px;}
#tz_73{left:162px;bottom:502px;letter-spacing:-0.2px;word-spacing:-0.45px;}
#t10_73{left:162px;bottom:485px;letter-spacing:-0.2px;word-spacing:1.5px;}
#t11_73{left:162px;bottom:468px;letter-spacing:-0.21px;word-spacing:3.08px;}
#t12_73{left:552px;bottom:468px;letter-spacing:-0.22px;word-spacing:3.09px;}
#t13_73{left:635px;bottom:468px;letter-spacing:-0.19px;word-spacing:3.08px;}
#t14_73{left:162px;bottom:451px;letter-spacing:-0.2px;word-spacing:1.2px;}
#t15_73{left:162px;bottom:435px;letter-spacing:-0.19px;word-spacing:0.03px;}
#t16_73{left:141px;bottom:395px;letter-spacing:0.14px;}
#t17_73{left:162px;bottom:361px;letter-spacing:0.12px;word-spacing:0.02px;}
#t18_73{left:162px;bottom:343px;letter-spacing:0.11px;word-spacing:0.02px;}
#t19_73{left:162px;bottom:309px;letter-spacing:-0.19px;word-spacing:-0.68px;}
#t1a_73{left:162px;bottom:293px;letter-spacing:-0.2px;word-spacing:0.13px;}
#t1b_73{left:162px;bottom:276px;letter-spacing:-0.2px;word-spacing:2.67px;}
#t1c_73{left:162px;bottom:259px;letter-spacing:-0.19px;word-spacing:-0.64px;}
#t1d_73{left:162px;bottom:242px;letter-spacing:-0.19px;word-spacing:0.44px;}
#t1e_73{left:162px;bottom:226px;letter-spacing:-0.2px;word-spacing:-0.7px;}
#t1f_73{left:162px;bottom:209px;letter-spacing:-0.2px;word-spacing:1.01px;}
#t1g_73{left:162px;bottom:192px;letter-spacing:-0.19px;word-spacing:0.47px;}
#t1h_73{left:162px;bottom:176px;letter-spacing:-0.2px;word-spacing:0.27px;}
#t1i_73{left:162px;bottom:159px;letter-spacing:-0.19px;word-spacing:-0.23px;}
#t1j_73{left:162px;bottom:142px;letter-spacing:-0.19px;word-spacing:0.02px;}

.s0_73{font-size:15px;font-family:TimesNewRomanPSMT_6e;color:#000;}
.s1_73{font-size:17px;font-family:Calibri_7k;color:#000;}
.s2_73{font-size:18px;font-family:TimesNewRomanPS-BoldMT_7w;color:#000;}
.s3_73{font-size:15px;font-family:TimesNewRomanPS-BoldMT_7s;color:#000;}
.s4_73{font-size:15px;font-family:TimesNewRomanPSMT_aw;color:#000;}
.s5_73{font-size:15px;font-family:TimesNewRomanPS-BoldMT_a-;color:#000;}
</style>
<!-- End inline CSS -->

<!-- Begin embedded font definitions -->
<style id="fonts73" type="text/css" >

@font-face {
	font-family: Calibri_7k;
	src: url("fonts/Calibri_7k.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7s;
	src: url("fonts/TimesNewRomanPS-BoldMT_7s.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_7w;
	src: url("fonts/TimesNewRomanPS-BoldMT_7w.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_a-;
	src: url("fonts/TimesNewRomanPS-BoldMT_a-.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_6e;
	src: url("fonts/TimesNewRomanPSMT_6e.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPSMT_aw;
	src: url("fonts/TimesNewRomanPSMT_aw.woff") format("woff");
}

</style>
<!-- End embedded font definitions -->

<!-- Begin page background -->
<div id="pg73Overlay" style="width:100%; height:100%; position:absolute; z-index:1; background-color:rgba(0,0,0,0); -webkit-user-select: none;"></div>
<div id="pg73" style="-webkit-user-select: none;"><object width="935" height="1210" data="73/73.svg" type="image/svg+xml" id="pdf73" style="width:935px; height:1210px; -moz-transform:scale(1); z-index: 0;"></object></div>
<!-- End page background -->


<!-- Begin text definitions (Positioned/styled in CSS) -->
<div class="text-container"><span id="t1_73" class="t s0_73">Brain: Chemistry to Cognition </span><span id="t2_73" class="t s1_73">65 </span>
<span id="t3_73" class="t s2_73">IL2.8. Advanced neuroimaging and artificial intelligence in understanding human brain </span>
<span id="t4_73" class="t s2_73">health and diseases: Sneaking into gliomas and aging brain </span>
<span id="t5_73" class="t s3_73">Vivek Tiwari </span>
<span id="t6_73" class="t s0_73">Department of Biological Sciences, Indian Institute of Science Education and Research (IISER), </span>
<span id="t7_73" class="t s0_73">Berhampur </span>
<span id="t8_73" class="t s4_73">Understanding the clinically relevant molecular underpinning of brain disorders is complicated, given the </span>
<span id="t9_73" class="t s4_73">challenges associated with surgical interventions in brain to obtain the tissue biopsies for genetic and epigenetic </span>
<span id="ta_73" class="t s4_73">investigations. Therefore, my group is employing advanced neuroimaging and artificial intelligence to investigate </span>
<span id="tb_73" class="t s4_73">clinically relevant hard-to-detect structural, vascular and biochemical events signifying the brain health, especially </span>
<span id="tc_73" class="t s4_73">in aging associated disorders and gliomas to bypass the need of invasive procedures. We are employing ‘Advanced </span>
<span id="td_73" class="t s4_73">neuroimaging, RadioGenomics and RadioMetabolomics’ approaches to understand aging, dementia and glioma </span>
<span id="te_73" class="t s4_73">associated ‘Quantitative signatures’ for improved clinical management. </span>
<span id="tf_73" class="t s4_73">In 2016, WHO revised the glioma classification and introduced for the first-time molecular basis of glioma </span>
<span id="tg_73" class="t s4_73">classification as IDH mutant and wild type. Irrespective of the histologic grade, the gliomas harboring somatic </span>
<span id="th_73" class="t s4_73">mutations in isocitrate dehydrogenase (IDH) confers better prognosis and improved survival compared to the IDH </span>
<span id="ti_73" class="t s4_73">wildtype. The structural and geometric heterogeneities across gliomas are potentially a phenotypic manifestation </span>
<span id="tj_73" class="t s4_73">of tumor relevant genetic and epigenetic events. Identification of mutational and epigenetic status requires tissue </span>
<span id="tk_73" class="t s4_73">biopsy/surgical interventions. Given the complexity of surgical interventions in the brain, clinical management of </span>
<span id="tl_73" class="t s4_73">gliomas and other brain tumor patients is challenging. Establishing the genome-phenome relationship by </span>
<span id="tm_73" class="t s4_73">unravelling the tumor substructural geometry; viz enhancing, non-enhancing, necrosis and edema components </span>
<span id="tn_73" class="t s4_73">across gliomas harboring distinct molecular background, is likely to establish a non-invasive method of structural </span>
<span id="to_73" class="t s4_73">geometry as an indicator of molecular status and associated clinical outcomes for prior treatment planning and </span>
<span id="tp_73" class="t s4_73">improved clinical management. To investigate the structural and molecular basis of genotypic status in gliomas, </span>
<span id="tq_73" class="t s4_73">we have performed glioma segmentation and fractal dimension measurements using MR images. The fractal </span>
<span id="tr_73" class="t s4_73">dimensions serve as a metric for quantifying the complexity and irregularity exhibited by various shapes and </span>
<span id="ts_73" class="t s4_73">structures. Fractal dimensions vary for IDH mutant tumors compared to IDH wild type tumors. A combined </span>
<span id="tt_73" class="t s4_73">approach of fractality and lacunarity of tumor subcomponents is predictive of IDH mutational status and MGMT </span>
<span id="tu_73" class="t s4_73">methylation. In addition to structural remodelling, my recent findings elucidate the alterations in pool and kinetics </span>
<span id="tv_73" class="t s4_73">of ‘Glycine’ and ‘2-Hydroxyglutarate’ as indicator of glioma aggressiveness and its role in patient survival. </span>
<span id="tw_73" class="t s4_73">Biochemical heterogeneity of gliomas is predictive of patient survival. </span>
<span id="tx_73" class="t s4_73">In a parallel study on aging and dementia, we observe that a subset of the aging population maintains the cognitive </span>
<span id="ty_73" class="t s4_73">abilities with minimal structural changes with aging while another subset of population transforms to mild </span>
<span id="tz_73" class="t s4_73">cognitive impairment and dementia with manifold structural variations. Most importantly is heterogeneity of white </span>
<span id="t10_73" class="t s4_73">matter lesions arising because of small vessel infarcts. With the application of advanced neuroimaging and AI </span>
<span id="t11_73" class="t s4_73">methods in aging and dementia patients, we establish that the </span><span id="t12_73" class="t s5_73">‘Brain Age’ </span><span id="t13_73" class="t s4_73">pattern is distinct from that of </span>
<span id="t14_73" class="t s4_73">Chronological age. For the first time we shoe that Only Three MRI features are that includes quantity of small </span>
<span id="t15_73" class="t s4_73">vessel pathologies are definitive of Brain age and Cognitive status. </span>
<span id="t16_73" class="t s2_73">IL2.9. Glutamate receptor trafficking: Ins and Outs </span>
<span id="t17_73" class="t s3_73">Samarjit Bhattacharyya </span>
<span id="t18_73" class="t s0_73">Department of Biological Science, Indian Institute of Science Education and Research (IISER), Mohali </span>
<span id="t19_73" class="t s4_73">In the brain, a variety of neurotransmitters and neuromodulators act on target receptors to activate cellular signaling </span>
<span id="t1a_73" class="t s4_73">events which transfer information from one cell to the next. Normal signaling depends on accurate localization of </span>
<span id="t1b_73" class="t s4_73">such receptors in specific regions of the cell, and the process of receptor trafficking plays a critical role in </span>
<span id="t1c_73" class="t s4_73">controlling this localization. In addition, in case of most G-protein-coupled receptors (GPCRs), receptor trafficking </span>
<span id="t1d_73" class="t s4_73">also plays crucial roles in the regulation of the receptor. Despite the obvious significance of this process, we still </span>
<span id="t1e_73" class="t s4_73">know very little about the molecular mechanisms that mediate trafficking of neurotransmitter receptors in the brain. </span>
<span id="t1f_73" class="t s4_73">Our labs specific interest lies in studying the cellular and molecular mechanisms that regulate the trafficking of </span>
<span id="t1g_73" class="t s4_73">glutamate receptors in the central nervous system. These trafficking events are thought to be critical for virtually </span>
<span id="t1h_73" class="t s4_73">all forms of experience-dependent plasticity, including learning and memory and are believed to play crucial role </span>
<span id="t1i_73" class="t s4_73">in various neuropsychiatric disorders. The talk will be focused on the trafficking of glutamate receptors in neurons </span>
<span id="t1j_73" class="t s4_73">and implications of these processes in the brain. </span></div>
<!-- End text definitions -->


</div>
</body>
</html>
